.scripts/bundlesize.js

```diff
@@ -23,26 +20,52 @@ function execVerbose(script) {
   return exec(script, { maxBuffer: 1024 * 1024 });
 }
 
-async function main() {
-  await execVerbose("git reset --hard " + process.env.TRAVIS_BRANCH);
-  dependencies.refreshNodeModules();
-  const baseSize = await getBundleSize();
-
-  await execVerbose("git reset --hard " + process.env.TRAVIS_PULL_REQUEST_SHA);
-  dependencies.refreshNodeModules();
-  const headSize = await getBundleSize();
-
-  const change = (headSize / baseSize) - 1;
-  const percentChange = (Math.abs(change) * 100).toFixed(2) + '%';
-
+/**
+ * Output a message to the console.
+ */
+function outputMessage(message) {
```

Comment:
Note that ci-sidecar will only consume the first `---output` block--subsequent blocks will be ignored